substrate/pkg/controller/substrate/cluster/addons/kitoperator.go

```diff
@@ -33,8 +33,8 @@ func (l *KITOperator) Create(ctx context.Context, substrate *v1alpha1.Substrate)
 	if err := helm.NewClient(*substrate.Status.Cluster.KubeConfig).Apply(ctx, &helm.Chart{
 		Namespace:       "kit",
 		Name:            "kit-operator",
-		Repository:      "https://github.com/awslabs/kubernetes-iteration-toolkit/releases/download/kit-operator-0.0.12",
-		Version:         "0.0.12",
+		Repository:      "https://github.com/awslabs/kubernetes-iteration-toolkit/releases/download/kit-operator-0.0.13",
```

Comment:
This requires us to release the new kit-operator helm chart with the above changes.

Reply:
done.